src/shared/views/ChangeDetection/Task.js

```diff
@@ -159,6 +159,8 @@ export default class ChangeDetectionTask extends React.PureComponent<
                             icon: 'fixme',
                         }}
                         boxType="fixme"
+                        bottomOffset="45%"
```

Comment:
not sure why you changed something in change detection here.

Reply:
see below